opennms-icmp/opennms-icmp-best/src/main/java/org/opennms/netmgt/icmp/best/BestMatchPingerFactory.java

```diff
@@ -85,7 +85,7 @@ private static PingerMatch tryPinger(final Class<? extends Pinger> pingerClass)
 
         try {
             final long timeout = Long.valueOf(System.getProperty("org.opennms.netmgt.icmp.best.timeout", "500"), 10);
-            final Number result = pinger.ping(InetAddressUtils.getLocalHostAddress(), timeout, 0);
+            final Number result = pinger.ping(InetAddressUtils.getLocalLoopbackAddress(), timeout, 0);
```

Comment:
If we use optionals above, then this can become:

```
pinger.ping(InetAddressUtils.getLocalLoopbackAddress()
                    .orElseThrow(() -> new IllegalStateException("No result pinging localhost.")), timeout, 0);
```